RandomCatsUITests/RandomCatsUITests.swift

```diff
@@ -1,13 +1,13 @@
 //
-//  RandomDogsUITests.swift
```

Comment:
If you want to I can teach you how to write UI tests for this project, Apple's automated testing is really cool and fun to use.